task/upload.go

```diff
@@ -99,7 +101,7 @@ func handleUploadVOD(p handleUploadVODParams) (*TaskHandlerOutput, error) {
 		}
 		glog.Infof("Processing upload vod catalyst callback. taskId=%s status=%q",
 			tctx.Task.ID, callback.Status)
-		if callback.Status != "success" {
+		if callback.Status != catalystClients.TranscodeStatusCompleted {
```

Comment:
Uhm, I think this change was in `main`. Do you have merges in your branch or sth?